### DISCUSSION TITLE
How to make application layout to be responsive.

### DISCUSSION THREAD

**Discussion by umeshvenkat** [blazority/support#45] [Q&A] [answered]
In mobile view no option provided to show-up the menu. on tablet view 3 dots appears which can trigger the menu.

**Comment by karan-kang** [comment, marked as answer]
Hi @umeshvenkat,

Would it be possible to share a snippet of how you are using the application layout?

**Reply by umeshvenkat** [reply]
I got it worked. Thanks.

**Reply by karan-kang** [reply]
Great, thanks!
